pynicotine/gtkgui/ui/settings/uploads.ui

```diff
@@ -380,7 +380,7 @@
                 <child>
                   <object class="GtkLabel">
                     <property name="visible">True</property>
-                    <property name="label" translatable="yes">Upload queue type:</property>
+                    <property name="label" translatable="yes">Free slot allocation:</property>
```

Comment:
The old string was more descriptive, imo.

Reply:
I disagree, "queue type" could seem to mean anything, and we already have "Queue Behavior" as the sub-section title.

The intention here is to associate this setting with the "Free Slot" mechanism, to help users understand that this section controls the behaviour of their Free Slots, i.e. the method by which their Free Slots are allocated.